src/commons/tracer.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * Helper file to whirl up the tracer class and export it for ease of use.
```

Comment:
May just be me, but I had no idea what "whirl up" meant 😅  could we maybe use "instantiate"?